templates/challenge.html

```diff
@@ -243,6 +268,14 @@
           fail type check, while others can pass.<br>
           Hit the "▶️ Run" button to see result.
         </p>
+        <div class="hints-container">
+          {% if hints_for_display %}
+            <a id="read-hints" role="button" class="secondary outline" href="javascript:void(0);">💡 Read Hints</a>
+          {% else %}
+            <div id="hints-missing" href="javascript:void(0);">💡 No Hints Available</div>
```

Comment:
This is fine, but I slightly prefer not showing anything if there's no hint

Reply:
I prefer to leave this text as it is, because in the future we may want to encourage users to submit hints or other types of feedback.

Reply:
okay